CondFormats/GeometryObjects/interface/PTrackerParameters.h

```diff
@@ -115,6 +115,8 @@ class PTrackerParameters
   TIBItem tib;
   TIDItem tid;
   TOBItem tob;
+
+  std::vector<int> pDetIdShifts;
```

Comment:
@davidlange6 The policy of not allowing schema evolution at all in condition objects was agreed between AlCa and DB when moving to CondDBv2. In CondDBv1, we were suffering schema evolution issues especially for ROOT streamed data members; in ORA, this was mitigated with some reflection properties when defining the ORM layer and/or using a new schema hosting the new shape.
Operationally, given that the need to avoid changes in production schemata (especially now given that we are still using the old schemata), this is fine.